source/extensions/common/workload_discovery/api.cc

```diff
@@ -46,10 +46,25 @@ Istio::Common::WorkloadMetadataObject convert(const istio::workload::Workload& w
   default:
     break;
   }
+
+  constexpr absl::string_view defaultNs = "default";
```

Comment:
can you pull them out of and use correct style, e.g. `constexpr absl::string_view DefaultNamespace = ...`

Reply:
Pull them out to where? I skimmed through the Envoy style guide and didn't see what the proper scoping was. I'll add them to the namespace, but let me know if they need to be elsewhere